2021-04-20 14:24:19,720 INFO    MainThread:6412 [wandb_setup.py:_flush():69] setting env: {}
2021-04-20 14:24:19,720 INFO    MainThread:6412 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-20 14:24:19,720 INFO    MainThread:6412 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210420_142419-3h3fhgyo/logs/debug.log
2021-04-20 14:24:19,720 INFO    MainThread:6412 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210420_142419-3h3fhgyo/logs/debug-internal.log
2021-04-20 14:24:19,721 INFO    MainThread:6412 [wandb_init.py:init():369] calling init triggers
2021-04-20 14:24:19,721 INFO    MainThread:6412 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-20 14:24:19,721 INFO    MainThread:6412 [wandb_init.py:init():418] starting backend
2021-04-20 14:24:19,721 INFO    MainThread:6412 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-20 14:24:19,732 INFO    MainThread:6412 [backend.py:ensure_launched():123] starting backend process...
2021-04-20 14:24:19,737 INFO    MainThread:6412 [backend.py:ensure_launched():128] started backend process with pid: 6430
2021-04-20 14:24:19,738 INFO    MainThread:6412 [wandb_init.py:init():423] backend started and connected
2021-04-20 14:24:19,739 INFO    MainThread:6412 [wandb_init.py:init():465] updated telemetry
2021-04-20 14:24:19,739 INFO    MainThread:6412 [wandb_init.py:init():484] communicating current version
2021-04-20 14:24:20,497 INFO    MainThread:6412 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-20 14:24:20,497 INFO    MainThread:6412 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-20 14:24:20,747 INFO    MainThread:6412 [wandb_init.py:init():522] starting run threads in backend
2021-04-20 14:24:20,776 INFO    MainThread:6412 [wandb_run.py:_console_start():1524] atexit reg
2021-04-20 14:24:20,777 INFO    MainThread:6412 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-20 14:24:20,777 INFO    MainThread:6412 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-20 14:24:20,781 INFO    MainThread:6412 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-20 14:24:20,781 INFO    MainThread:6412 [wandb_init.py:init():546] run started, returning control to user process
2021-04-20 14:24:30,756 INFO    MainThread:6412 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-20 14:24:30,757 INFO    MainThread:6412 [wandb_run.py:_restore():1466] restore
2021-04-20 14:24:32,973 INFO    MainThread:6412 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 539
  total_bytes: 539
}

2021-04-20 14:24:33,085 INFO    MainThread:6412 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 539
  total_bytes: 539
}

2021-04-20 14:24:33,769 INFO    MainThread:6412 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 539
  total_bytes: 3393
}

2021-04-20 14:24:34,485 INFO    MainThread:6412 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3393
  total_bytes: 3393
}

2021-04-20 14:24:34,587 INFO    MainThread:6412 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3393
  total_bytes: 3393
}

2021-04-20 14:24:34,689 INFO    MainThread:6412 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3393
  total_bytes: 3393
}

2021-04-20 14:24:34,791 INFO    MainThread:6412 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3393
  total_bytes: 3393
}

2021-04-20 14:24:34,893 INFO    MainThread:6412 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3393
  total_bytes: 3393
}

2021-04-20 14:24:35,994 INFO    MainThread:6412 [wandb_run.py:_show_files():1838] logging synced files
